test/parallel/test-dgram-setTTL.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const dgram = require('dgram');
+const socket = dgram.createSocket('udp4');
+
+socket.bind(common.PORT);
+socket.on('listening', function() {
+  var result = socket.setTTL(16);
+  assert.equal(result, 16);
```

Comment:
nit: could be `assert.strictEqual(socket.setTTL(16), 16);` Or if that's less readable, another `const`. :)

Also, isn't it better to use `strictEqual` for this kind of stuff?